tests/config/bug_1758.cfg

```diff
@@ -0,0 +1,11 @@
+output_tab_size                 = 4
+sp_arith                        = force
+sp_assign                       = force
+sp_compare                      = force
+sp_before_sparen                = force
+indent_columns                  = 4
+nl_end_of_file                  = force
+nl_end_of_file_min              = 2
```

Comment:
Still not to happy about the config, some of those options are definitely not needed. This needs a alteration of the test file. Id also like to see a test case where we have something like: `while(){if(){}}`

Reply:
Why not "happy"?
This is the output of option_reducer.py

Reply:
Yes the reduction script tries to reduce the option without changing the output formatting. The problem in this case is that the output file itself is to complex, it need to be simplified.

The initial problem is that the for braces are disappearing when they should not.

That means we definitely do not need any of:
```sql
sp_*
nl_end_of_file                  = force
nl_end_of_file_min              = 2
```

we most likely do not need:
```sql
output_tab_size                 = 4
indent_columns                  = 4
```

and  _maybe_ we do need:
```sql
nl_for_brace                    = remove
nl_after_if                     = force
mod_full_brace_if               = remove
```